### PR TITLE
Feature/uweb 5248 brands animation

### PR DESCRIPTION
**Pull request by dan-entous**
* [x] add brands component
* [x] add brand component
* [x] add list-unordered-limit component
* [x] correct refresh icon so it uses currentColor
* [x] add step row animation

### CODE REVIEW

File: src/js/components/molecules/brand/stories/brand.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Brand
```
Comment (zwaardje) [inline src/js/components/molecules/brand/stories/brand.md]: Update to the correct links?

File: src/js/components/organisms/brands/brands.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export class Brands extends Vue implements BrandsProps {
+    @Prop({ type: Object }) ctas?: BrandsProps['ctas'];
+    @Prop({ type: Object }) expand?: BrandsProps['expand'];
+    @Prop({ type: Array, required: true }) items!: BrandsProps['items'];
+    @Prop({ type: Object }) limit?: BrandsProps['limit'];
+    @Prop({ type: String }) subtitle?: BrandsProps['subtitle'];
+    @Prop({ type: String, required: true }) title!: BrandsProps['title'];
+
+    get listItems(): object {
```
Comment (zwaardje) [inline src/js/components/organisms/brands/brands.ts]: Why do something the IS should do. We should not be creating our own types. Just stick to items. In your html your are already defining that you are using a ul. then just use the <LiItem />

File: src/js/components/organisms/brands/brands2.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            justify-content: center;
+            list-style: none;
+
+            margin-top: -$buildingUnit;
+            margin-bottom: 0;
+            margin-left: auto;
+            margin-right: auto;
+            padding-left: $buildingUnit;
+
+            max-width: 6 * (120 + $buildingUnit) + $buildingUnit;
```
Comment (zwaardje) [inline src/js/components/organisms/brands/brands2.scss]: Changing the image height will kill the row animation. Maybe convert the magic number to a constant

File: src/js/components/atoms/li-item/template.vue.html [hunk 4]
```diff
@@ -1,4 +1,5 @@
 <li class="list__item">
-    <template v-if="!item.item">{{ item }}</template>
-    <template v-else><component :is="item.type" v-bind="item.item"/></template>
+    <template v-if="item.item"><component :is="item.type" v-bind="item.item"/></template>
```
Comment (zwaardje) [inline src/js/components/atoms/li-item/template.vue.html]: This does not effect the first initial server side render? For the crawler magic to happen.

File: src/js/components/molecules/list-unordered-limit/list-unordered-limit.ts [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+import { default as ViewportUtil } from 'utils/viewport';
+import WithRender from './list-unordered-limit.vue.html';
+
+@WithRender
+@Component({
+    name: 'ListUnorderedLimit',
+})
+
+export class ListUnorderedLimit extends Mixins(StepRowAnimation) implements ListUnorderedLimitProps {
+    @Prop({ type: Object, required: true }) listItems!: { items: [string[] | object[]], type: string };
```
Comment (zwaardje) [inline src/js/components/molecules/list-unordered-limit/list-unordered-limit.ts]: We should not be creating our own types.

File: src/js/components/molecules/list-unordered/template.vue.html [hunk 6]
```diff
@@ -1,6 +1,7 @@
 <ul class="list">
     <template v-for="(item, index) in listItems.items">
         <template v-if="!item.item"><component :is="listItems.type" :item="item" :key="index"/></template>
```
Comment (zwaardje) [inline src/js/components/molecules/list-unordered/template.vue.html]: This does not effect the first initial server side render? For the crawler magic to happen. Vue could also get a mismatch in html content and fail on rendering.

File: src/js/components/molecules/brand/stories/brand.md (hunk 1 above)
Comment (dan-entous) [reply src/js/components/molecules/brand/stories/brand.md]: resolved in discussion with @zwaardje

File: src/js/components/organisms/brands/brands.ts (hunk 2 above)
Comment (dan-entous) [reply src/js/components/organisms/brands/brands.ts]: resolved in discussion with @zwaardje

File: src/js/components/atoms/li-item/template.vue.html (hunk 4 above)
Comment (dan-entous) [reply src/js/components/atoms/li-item/template.vue.html]: resolved in discussion with @zwaardje

File: src/js/components/molecules/list-unordered-limit/list-unordered-limit.ts (hunk 5 above)
Comment (dan-entous) [reply src/js/components/molecules/list-unordered-limit/list-unordered-limit.ts]: resolved in discussion with @zwaardje

File: src/js/components/molecules/list-unordered/template.vue.html (hunk 6 above)
Comment (dan-entous) [reply src/js/components/molecules/list-unordered/template.vue.html]: resolved in discussion with @zwaardje

**Pull request merged**